src/Microsoft.Framework.Configuration.Ini/Ini/IniConfigurationSource.cs

```diff
@@ -110,7 +110,7 @@ internal void Load(Stream stream)
                     if (line[0] == '[' && line[line.Length - 1] == ']')
                     {
                         // remove the brackets
-                        sectionPrefix = line.Substring(1, line.Length - 2) + ":";
+                        sectionPrefix = line.Substring(1, line.Length - 2) + Constants.KeyDelimiter[0];
```

Comment:
You don't need `[0]`. line.Substring returns a string, the result is a string so you can just add another string.